features/git-town-kill/supplied_branch/feature_branch/with_tracking_branch.feature

```diff
@@ -21,8 +21,11 @@ Feature: git town-kill: killing the given feature branch
     Then it runs the commands
       | BRANCH       | COMMAND                       |
       | good-feature | git fetch --prune             |
+      |              | git add -A                    |
+      |              | git stash                     |
```

Comment:
We don't want to preserve the open changes when killing a branch. The open changes are considered a part of the branch to kill, and should be thrown away together with it. Here is the usage scenario:
- I have an idea for something
- I run `git hack` to create a new branch
- I try the idea out and it turns it it leads nowhere
- I run `git kill` to get rid of the entire thing

I am not sure whether there is a meaningful Git history after running `git kill`.  The "previous" branch is gone now. We could simply not wrap `git kill` at all.